GPU/GPUTracking/Global/GPUChainTrackingClusterizer.cxx

```diff
@@ -612,37 +612,62 @@ int32_t GPUChainTracking::RunTPCClusterizer(bool synchronizeOutput)
   }
 
 #ifdef GPUCA_HAS_ONNX
+  const GPUSettingsProcessingNNclusterizer& nn_settings = GetProcessingSettings().nn;
```

Comment:
With the code in GPUChainTrackingClusterizer.cxx growing, we could think about moving all NN code out into a GPUChainClusterizerNN.cxx